### PR TITLE
Implement points module

### PR DESCRIPTION
**Pull request by Mahmoudzaid2994**
### Jira URL

https://optimumpartners.atlassian.net/browse/JIRA_ISSUE_ID

### Description

_What problem is this branch meant to solve, what changes does it make, and how do these changes fix that problem?_

### Dependencies in Other PRs,Repo

_If this pull request relates to or depends on any changes in backend/frontend repo PR link it here._

### Testing Instructions

_How to test this and check for any possible regressions. Provide test data when applicable._

### CODE REVIEW

File: src/services/action-types.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  closeDeal: 'Close deal',
+  completeProfile: 'Complete profile',
+  completeRegistration: 'Registration',
+  createNewListing: 'New listing',
+  dailyLogin: 'Login',
+});
+
+export default {
+  actionTypes,
+};
```
Comment (mohdabed123) [inline src/services/action-types.js:14]: there is a file for constants in utils that we can put actionTypes in there

File: src/components/notifications/notification.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      tabIndex="0"
+      onAnimationEnd={onAnimationEnd}
+      className={show ? 'in' : 'out'}
+      role="alert"
+    >
+      <NotificationMessage>
+        <NotificationSvg>
+          <NotificationIcon size="16" />
+        </NotificationSvg>
+        <Message> You&#39;ve earned {points} points !</Message>
```
Comment (mohdabed123) [inline src/components/notifications/notification.jsx:53]:
```suggestion
        <Message> You have earned {points} points !</Message>
```
no need for apostrophe

**Review by mohdabed123 — COMMENTED** [review]
Small comments

File: src/services/action-types.js (hunk 1 above)
Comment (Mahmoudzaid2994) [reply src/services/action-types.js]: done

File: src/components/notifications/notification.jsx (hunk 2 above)
Comment (Mahmoudzaid2994) [reply src/components/notifications/notification.jsx:53]: this copy from the Figma I changed `'` with `&#39;` because it's return eslint error

**Review by mohdabed123 — APPROVED** [review]

**Pull request merged**